lib/ansible/cli/galaxy.py

```diff
@@ -197,7 +197,10 @@ def api(self):
 
 
 class GalaxyCLI(CLI):
-    '''command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.'''
+    '''Command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.
```

Comment:
The "roles" bit longer seems accurate since it manages collections too. Though I suppose fixing that may be fine in a separate PR.
Additionally, it'd be nice to follow PEP 257, having a short title, an empty line and a multi-line description with full sentences.

Reply:
Heh, I didn't even notice it only mentions roles. Good catch.

Reply:
Fixed the formatting, made the title simpler. Feel free to suggest a better description.